app/main.py

```diff
@@ -1,25 +1,65 @@
-from abc import ABC
+from typing import Any, Type
```

Comment:
The import statement is incorrect. The correct syntax should be `from typing import Any, Type` without the slash.